vehicle/fiat/controller.go

```diff
@@ -100,3 +100,18 @@ func (c *Controller) disableConflictingChargeSchedule(schedule *Schedule) {
 		schedule.EnableScheduleType = false
 	}
 }
+
+var _ api.Resurrector = (*Controller)(nil)
+
+func (c *Controller) WakeUp() error {
+	if c.pin == "" {
+		return api.ErrMissingCredentials
```

Comment:
This will always return an error in the log. If the user doesn't enter a PIN I'd treat this as "user decision" and not error here, hence silently return nil or `api.ErrNotAvailable` (which usually gets ignored). Wdyt?

Reply:
It would also good to mention then in the template that wakeup requires the pin.